.gitignore

```diff
@@ -60,3 +60,4 @@ $RECYCLE.BIN/
 Network Trash Folder
 Temporary Items
 .apdisk
+*.csv#
```

Comment:
Why modify the .gitignore? 
Are the csv's in the base directory? Why the '#' at the end?

Reply:
.extension# is used by OpenOffice in its efforts to prevent external modifications to files. It doesn't always clear them properly.

The csvs are where they were originally..?

Reply:
Ehhhhh, I guess that's acceptable. (Ideally, one would just close OpenOffice prior to something careless like `git add *` (which should be avoided in the first place), but I suppose this does no harm)